services/database/index.ts

```diff
@@ -1,5 +1,16 @@
 import { PrismaClient } from "@prisma/client";
 
-const prisma = new PrismaClient();
+interface CustomGlobal extends NodeJS.Global {
+    prisma: PrismaClient;
+}
+
+// Prevent multiple instances of Prisma Client in development
+declare const global: CustomGlobal;
+
+const prisma = global.prisma || new PrismaClient();
```

Comment:
what's this line doing here?

Reply:
![image](https://user-images.githubusercontent.com/46411197/123666039-b5aa7400-d7ed-11eb-9c0d-37532fc0700b.png)
https://next-auth.js.org/adapters/prisma-legacy
This is to prevent multiple instances of the prisma client in development due to hot reloading